<img src="https://raw.githubusercontent.com/andre-marcos-perez/ebac-course-utils/main/media/logo/newebac_logo_black_half.png" alt="ebac-logo">

---

# **Módulo** | Python: Variáveis & Tipos de Dados
Caderno de **Exercícios**<br> 
Professor [André Perez](https://www.linkedin.com/in/andremarcosperez/)

---

# **Tópicos**

<ol type="1">
  <li>Introdução ao Google Colab;</li>
  <li>Variáveis;</li>
  <li>Números;</li>
  <li><i>Strings;</i></li>
  <li>Boleanos.</li>
</ol>

---

# **Exercícios**

## 1\. Google Colab

Crie uma célula de código que escreva o texto "Olá mundo!", utilize o comando `print`.

Crie uma célua de texto e adicione uma imagem.

In [1]:
print("Olá mundo!");

Olá mundo!


![imagem](Imagem.webp)

---

## 2\. Números

Preencha as células de código para preencher os valores de (A), (B) e (C) na tabela de ticket médio abaixo:

<br>

| Dia   | Valor Total Vendas | Qtd Total Vendas | Ticket Medio |
|-------|--------------------|------------------|--------------|
| 19/01 | (A)                | 3                | 320.52       |
| 20/01 | 834.47             | (B)              | 119.21       |
| 23/01 | 15378.12           | 5                | (C)          |

In [3]:
# (A)
A = 320.52 / 3
print(A)

106.83999999999999


In [4]:
# (B)
B = 834.47 / 119.21
print(B)

7.000000000000001


In [5]:
# (C)
C = 15378.12 * 5
print(C)

76890.6


Resultado: 

| Dia   | Valor Total Vendas | Qtd Total Vendas | Ticket Medio |
|-------|--------------------|------------------|--------------|
| 19/01 | 106.84             | 3                | 320.52       |
| 20/01 | 834.47             | 7                | 119.21       |
| 23/01 | 15378.12           | 5                | 76890.6      |

---

## 3\. Strings

Aplique três métodos distintos na *string* abaixo, você pode conferir alguns métodos neste [link](https://www.w3schools.com/python/python_ref_string.asp):

In [12]:
cancao = 'Roda mundo, roda gigante, roda moinho, roda pião.'

In [13]:
# Split - Divide a string pelo caracter indicado
cancao_split = cancao.split(',')
print(cancao_split)

# Capitaliza - Coloca a primeira letra em maiúsculo
cancao_capitalize = cancao.capitalize()
print(cancao_capitalize)

# Count - Conta a incidencia do caracter indicado
cancao_count = cancao.count('a')
print(cancao_count)

['Roda mundo', ' roda gigante', ' roda moinho', ' roda pião.']
Roda mundo, roda gigante, roda moinho, roda pião.
5


Extraia da string abaixo o valor da taxa **selic** na variável `selic` e o valor do **ano** na variavel `ano`. Imprima os valores na tela.

In [16]:
noticia = 'Selic vai a 2,75% e supera expectativas; é a primeira alta em 6 anos.'

In [23]:
find_selic = noticia.split()[3]
find_ano = noticia.split()[12]
print(f"Taxa Selic: {find_selic}")
print(f"Ano: {find_ano}")

Taxa Selic: 2,75%
Ano: 6


---

## 4\. Booleanos

Utilize a tabela da verdade para responder: qual o valor da variável x?

In [24]:
a = False
b = True

x = not a & b

In [25]:
# o valor de 'x' será o oposto do resultado de x é do y, sendo assim x e igual a True
print(x)   

True
